README.md

```diff
@@ -55,10 +55,17 @@ Scans PHP and JavaScript files, as well as theme stylesheets for translatable st
 	[--ignore-domain]
 		Ignore the text domain completely and extract strings with any text domain.
 
-	[--merge[=<file>]]
-		Existing POT file file whose content should be merged with the extracted strings.
+	[--merge[=<files>]]
+		One or more existing POT files whose contents should be merged with the extracted strings.
 		If left empty, defaults to the destination POT file.
 
+	[--except=<files>]
```

Comment:
As already commented, I think `--subtract` would be a clearer name.
Also, syntax for multiples files should be documented.